corehq/apps/export/filters.py

```diff
@@ -0,0 +1,114 @@
+from corehq.apps.es import filters as esfilters
```

Comment:
how do you see these filters being used? just wanted to know how this would differ than writing ES filters in `corehq/apps/es/export.py`